Code/Engine/GameEngine/XR/XRInterface.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
```

Comment:
I had it there before but it exploded over time and it will only get worse so I thought I'd just put all of them under XR. Also most of these interface will have shared base implementations anyways so they aren't pure interfaces to begin with. At the moment we have ezXRInterface, ezXRInputDevice, ezWindowXR, ezWindowOutputTargetXR, ezActorPluginWindowXR all with varying amount of base code already. Additionally I will need to add an anchor interface to support anchor components and spatial mapping support.